alembic/versions/14d1424b47_.py

```diff
@@ -0,0 +1,38 @@
+"""creates rsvp, adds, allow_rsvp to proposal_space
+
+Revision ID: 14d1424b47
+Revises: 1c496c114b6
+Create Date: 2015-01-30 16:09:42.434798
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '14d1424b47'
+down_revision = '1c496c114b6'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('rsvp',
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('proposal_space_id', sa.Integer(), nullable=False),
+    sa.Column('rsvp_action', sa.Enum('M', 'N', 'Y', name='rsvp_action'), nullable=True),
+    sa.ForeignKeyConstraint(['proposal_space_id'], ['proposal_space.id'], ),
+    sa.ForeignKeyConstraint(['user_id'], ['user.id'], ),
+    sa.PrimaryKeyConstraint('user_id', 'proposal_space_id'),
```

Comment:
Your primary key order should be `('proposal_space_id', 'user_id')` as you'll be looking up RSVPs from the ProposalSpace table most of the time (SQL indexes are left to right). Also reverse the order of the columns above.